analysis/02_new_model_input/05.0_relative_wealth_index.md

```diff
@@ -0,0 +1,58 @@
+# Relative Wealth Index
+
+The data is from `https://data.humdata.org/dataset/relative-wealth-index`
+
+```python
+%load_ext jupyter_black
+import pandas as pd
+import geopandas as gpd
+from pathlib import Path
+import os
+```
+
+```python
+input_dir = (
+    Path(os.getenv("STORM_DATA_DIR"))
+    / "analysis/02_new_model_input/05_vulnerablility/input/"
```

Comment:
I would make a variable like `base_dir = Path(os.getenv("STORM_DATA_DIR")) / "analysis/02_new_model_input` and then use that instead of writing the full file path each time